.gitignore

```diff
@@ -25,6 +25,7 @@ SHA256SUMS*
 __debug_bin
 id_tx_io.json
 .idea
+.vscode
```

Comment:
why?
we are changing this to make vscode developers life better :P

Reply:
hmm because why would we commit stuff like `dryrun.log` and `configurationCache.log`?
if we want we can upload `launch.json` specifically

Reply:
Maybe we need to ignore .vscode/*.log

Reply:
Then gitignore `dryrun.log` * `configurationCache.log`

Reply:
We've added debug configs, linting and formatting for go, and rust-analyzer config for the enclave so now it's 1000x better than CLion.

Reply:
also lol:
https://github.com/scrtlabs/SecretNetwork/pull/1085/files#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947L22